<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PortInitialization</class>
 <widget class="QWidget" name="PortInitialization">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>550</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>端口号:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>30</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>波特率:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="PortNum">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="BaudRate">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>60</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>460800</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="OpenPort">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>打开串口</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ClosePort">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭串口</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ClearMessage">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>清除数据</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setVoltage">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>设置高压</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>270</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>270</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="returnData">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>259</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>串口状态:</string>
   </property>
   <widget class="QLabel" name="return_data">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="voltageIn">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>电压状态：</string>
   </property>
   <widget class="QCheckBox" name="Power_Set">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>关</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
